meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -58,7 +58,7 @@ FILES_${PN}-get = "${bindir}/aktualizr-get"
 FILES_${PN}-lite = "${bindir}/aktualizr-lite"
 
 # Force same RDEPENDS, packageconfig rdepends common to both
-RDEPENDS_${PN}-lite = "${RDEPENDS_aktualizr}"
+RDEPENDS_${PN}-lite = "${RDEPENDS_aktualizr} skopeo"
 
```

Comment:
probably have to check with @ricardosalveti, but I think you also need a:
```
RDEPENDS_${PN}-lite-lib = "${RDEPENDS_aktualizr} skopeo"
```

Reply:
most likely, if a custom client relying on aklite-lib uses Apps.

Reply:
@ricardosalveti I guess it should be something like :) ?
```
RDEPENDS_${PN}-lite-lib = "${RDEPENDS_aktualizr-lib} skopeo"
```